packages/flat-components/src/components/PeriodicRoomPage/index.less

```diff
@@ -2,6 +2,7 @@
     width: 100%;
     height: 100%;
```

Comment:
remove height can solve this bug

Reply:
yes, you're right. And the `width` property is redundant as well. What do you think?

Reply:
cc: @Cheerego7

Reply:
if you want fix this overflow bug refer to PR #1398 and the page layout of the web is different from the renderer-app, I will fix this in a moment

Reply:
OK